doc/sphinx/addendum/nsatz.rst

```diff
@@ -32,13 +40,36 @@ Nsatz: tactics for proving equalities in integral domains
 
    doing automatic introductions.
 
-   You can load the ``Nsatz`` module with the command ``Require Import Nsatz``.
+   `radicalmax`
+     bound when searching for r such that
+     :math:`c (P−Q) r = \sum_{i=1..s} S_i (P i − Q i)`.
+     This argument must be of type `N` (binary natural numbers).
+
+   `strategy`
+     gives the order on variables :math:`X_1,\ldots,X_n` and the strategy
+     used in Buchberger algorithm (see :cite:`sugar` for details):
+
+       * `strategy := 0%Z`: reverse lexicographic order and newest s-polynomial.
+       * `strategy := 1%Z`: reverse lexicographic order and sugar strategy.
+       * `strategy := 2%Z`: pure lexicographic order and newest s-polynomial.
+       * `strategy := 3%Z`: pure lexicographic order and sugar strategy.
+
+   `parameters`
+     a list of lists, each of type `@list R`, containing the variables :math:`X_{i_1},\ldots,X_{i_k}` among
+     :math:`X_1,\ldots,X_n` which are considered as parameters: computation will be performed with
+     rational fractions in these variables, i.e. polynomials are considered
+     with coefficients in :math:`R(X_{i_1},\ldots,X_{i_k})`. In this case, the coefficient
+     :math:`c` can be a non constant polynomial in :math:`X_{i_1},\ldots,X_{i_k}`, and the tactic
+     produces a goal which states that :math:`c` is not zero.
+
+   `variables`
+     a list of lists, each of type `@list R`, containing the variables in the decreasing order in
+     which they will be used in the Buchberger algorithm. If `variables` = :g:`(@[] R)`,
```

Comment:
This syntax `@` before the empty list notation `[]` is not supported, we should stick to `@nil R`.
```suggestion
     which they will be used in the Buchberger algorithm. If `variables := (@nil R)`,
```